ush/templates/FV3LAM_wflow.xml

```diff
@@ -2342,6 +2317,78 @@ MODULES_RUN_TASK_FP script.
   </task>
 
 {%- endif %}
+
+{%- if do_save_input %}
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="&SAVE_INPUT_TN;_prod{{ uscore_ensmem_name }}" cycledefs="prodcyc" maxtries="{{ maxtries_save_input }}">
+
+    &RSRV_DEFAULT;
+    &WALL_LIMIT_SAVE_RESTART;
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&SAVE_RESTART_TN;" "&JOBSDIR;/JREGIONAL_SAVE_INPUT"</command>
+    <nodes>{{ nnodes_save_restart }}:ppn={{ ppn_save_restart }}</nodes>
+    <walltime>{{ wtime_save_restart }}</walltime>
+    <nodesize>&NCORES_PER_NODE;</nodesize>
+    <jobname>&TAG;_&SAVE_INPUT_TN;{{ uscore_ensmem_name }}</jobname>
+    <join><cyclestr>&LOGDIR;/&SAVE_INPUT_TN;{{ uscore_ensmem_name }}_@Y@m@d@H.log</cyclestr></join>
+
+    <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+    <envar><name>PDY</name><value><cyclestr>@Y@m@d</cyclestr></value></envar>
+    <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>NWGES_DIR</name><value><cyclestr>&NWGES_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>SLASH_ENSMEM_SUBDIR</name><value><cyclestr>{{ slash_ensmem_subdir }}</cyclestr></value></envar>
+    <envar><name>cyc</name><value><cyclestr>@H</cyclestr></value></envar>
+    <envar><name>CYCLE_TYPE</name><value><cyclestr>prod</cyclestr></value></envar>
+
+    <dependency>
```

Comment:
Let's say this only for ensemble da part. We may want to simplify the dependence.